include/grpc++/impl/codegen/async_stream.h

```diff
@@ -225,7 +275,8 @@ class ClientAsyncWriter final : public ClientAsyncWriterInterface<W> {
   Call call_;
   CallOpSet<CallOpSendInitialMetadata> init_ops_;
   CallOpSet<CallOpRecvInitialMetadata> meta_ops_;
-  CallOpSet<CallOpSendMessage> write_ops_;
+  CallOpSet<CallOpSendInitialMetadata, CallOpSendMessage, CallOpClientSendClose>
+      write_ops_;
   CallOpSet<CallOpClientSendClose> writes_done_ops_;
   CallOpSet<CallOpRecvInitialMetadata, CallOpGenericRecvMessage,
```

Comment:
You mean merge read op (i.e RecvInitialMetdata) with write ops on client side?

I think for ServerAsyncReader we may merge meta_ops with finish_ops, and for ServerAsyncWriter and ServerAsyncReaderWriter we may merge meta_ops with write_ops